Cue/app/actions/library.js

```diff
@@ -110,7 +110,15 @@ async function syncDeck(change) : PromiseAction {
       }
     }
   } else if (change.action === "edit") {
-    serverDeck = await LibraryApi.editDeck(change)
+    if (change.name || change.tags || change.public
+      || (change.cards && change.cards.find(card => card.front || card.back || card.poistion))) {
```

Comment:
`position` and oof, this seems really fragile.

What happens if there are some cards which are edited and some which are flagged? Would it be better to introduce a new client-only action type `flag` to clearly separate the two actions?